homeassistant/components/light/mqtt.py

```diff
@@ -53,6 +55,8 @@
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
     vol.Optional(CONF_BRIGHTNESS_SCALE, default=DEFAULT_BRIGHTNESS_SCALE):
         vol.All(vol.Coerce(int), vol.Range(min=1)),
+    vol.Optional(CONF_NATURAL_BEHAVIOR, default=DEFAULT_NATURAL_BEHAVIOR):
```

Comment:
I don't like this to be an extra option. If we implement this we should implement it. Not allow people to disable it.

I did explore this path before and you will need to add support for an MQTT light that does not have a state topic, just a brightness topic. This should work as expected for `turn_on`, `turn_off` and `brightness_received`.

This should also be accompanied by tests.